Import `sp_runtime::generic::Era`

From the Substrate Technical channel:

> Hello @here,
> I am tring to run light clint code :-
> ```
> 
> 
> use substrate_api_client::{Api, node_metadata::Metadata, XtStatus,compose_extrinsic,extrinsic::xt_primitives::UncheckedExtrinsicV4};
> use sp_core::sr25519;
> use keyring::AccountKeyring;
> use sp_core::crypto::Pair;
> 
> 
> fn main() {
>     // instantiate an Api that connects to the given address
>     let url = "127.0.0.1:9944";
>     // if no signer is set in the whole program, we need to give to Api a specific type instead of an associated type
>     // as during compilation the type needs to be defined.
>     let from = AccountKeyring::Alice.pair();
>     let mut api = Api::new(format!("ws://{}", url)).set_signer(from);
>     let xt: UncheckedExtrinsicV4<_> = compose_extrinsic!( api.clone(),"System", "create_claim");
>     println!("[+] Composed Extrinsic:\n {:?}\n", xt);
>     let signer = AccountKeyring::Alice.pair();
>     api.signer = Some(signer);
>     let tx_hash = api.send_extrinsic(xt.hex_encode(), XtStatus::Finalized).unwrap();
>     println!("[+] Transaction got finalized. Hash: {:?}", tx_hash);
> 
> }
> ```
> 
> But getting error :-
> ```
> error[E0433]: failed to resolve: use of undeclared type or module `sp_runtime`
>   --> src/main.rs:14:39
>    |
> 14 |     let xt: UncheckedExtrinsicV4<_> = compose_extrinsic!( api.clone(),"System", "create_claim");
>    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `sp_runtime`
>    |
>    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
> 
> error[E0433]: failed to resolve: use of undeclared type or module `Era`
>   --> src/main.rs:14:39
>    |
> 14 |     let xt: UncheckedExtrinsicV4<_> = compose_extrinsic!( api.clone(),"System", "create_claim");
>    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `Era`
>    |
>    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
> 
> 
> ```

Cause by: https://github.com/scs/substrate-api-client/blob/ee31f4e6d2004ed9b841f31011516d81b23f6f11/src/extrinsic/mod.rs#L71

Some better re-exporting will solve this

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "substrate-api-client"
-version = "0.4.7-sub2.0.0-rc5"
+version = "0.4.8-sub2.0.0-rc5"
 authors = ["Supercomputing Systems AG <info@scs.ch>"]
 edition = "2018"
 

--- a/src/extrinsic/mod.rs
+++ b/src/extrinsic/mod.rs
@@ -69,8 +69,8 @@ macro_rules! compose_extrinsic_offline {
     $genesis_or_current_hash: expr,
     $runtime_spec_version: expr,
     $transaction_version: expr) => {{
-        use sp_runtime::generic::Era;
         use $crate::extrinsic::xt_primitives::*;
+        use $crate::sp_runtime::generic::Era;
         let extra = GenericExtra::new($era, $nonce);
         let raw_payload = SignedPayload::from_raw(
             $call.clone(),

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -80,6 +80,8 @@ use events::{EventsDecoder, RawEvent, RuntimeEvent};
 #[cfg(feature = "std")]
 use sp_runtime::{generic::SignedBlock, AccountId32 as AccountId, MultiSignature};
 
+pub extern crate sp_runtime;
+
 pub use sp_core::H256 as Hash;
 
 /// The block number type used in this runtime.

